plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestTrinoS3FileSystemAwsS3.java

```diff
@@ -28,14 +28,14 @@
 public class TestTrinoS3FileSystemAwsS3
         extends BaseTestTrinoS3FileSystemObjectStorage
 {
-    private String bucketName;
-    private String s3Endpoint;
+    private final String bucketName;
+    private final String s3Endpoint;
 
-    @BeforeClass
-    public void setup()
+    @Parameters({"s3.bucket", "s3.bucket-endpoint"})
+    public TestTrinoS3FileSystemAwsS3(String bucketName, String s3Endpoint)
     {
-        bucketName = requireNonNull(System.getenv("S3_BUCKET"), "Environment S3_BUCKET was not set");
```

Comment:
> Use testng Parameters rather than environment variables

the previous had advantage that surefire isn't masking test instantiation errors

anyway, this should come coupled with ci.yml changes, but this commit changes test only

Reply:
Don't these couple lines provide the params? https://github.com/trinodb/trino/pull/17563/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR335-R337

Reply:
not feeling strongly about this

Reply:
I have removed this commit for now, we can refactor elsewhere if we want to.